addons/base_setup/static/src/xml/res_config_edition.xml

```diff
@@ -6,7 +6,7 @@
         <div class="col-12 o_setting_box" id="edition">
             <div class="o_setting_right_pane">
                 <div class="user-heading">
-                    <h3>
+                    <h3 class="ml-0 pl-0">
```

Comment:
Seems like there is only padding on the element so the padding should be sufficient (idem for enterprise). It could also be replace by `px-0` to do the same on each side.
The `.app_settings_block h3` already set the margin to 0 everywhere

Reply:
Thanks for the review. I have followed your proposition.